x11-misc/clipnotify/clipnotify-1.0.2.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Notify on new X clipboard events"
+HOMEPAGE="https://github.com/cdown/clipnotify"
+SRC_URI="https://github.com/cdown/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Unlicense"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	x11-libs/libX11
+	x11-libs/libXfixes
+"
+RDEPEND="${DEPEND}"
+BDEPEND="${DEPEND}"
```

Comment:
DEPEND and BDEPEND packages are built for different architectures when cross-compiling

Reply:
@DarthGandalf: I'm not entirely sure what's the expected action in response to that remark from my side. What would you suggest using instead that would be acceptable?

Reply:
Please review https://devmanual.gentoo.org/general-concepts/dependencies/index.html and decide what dependency goes to where. Perhaps, by removing BDEPEND if the build system doesn't need anything installed.